swift-sdk/IterableMessaging.swift

```diff
@@ -121,47 +156,57 @@ public final class IterableInAppMessage : NSObject {
     /// the campaign id for this message
     public let campaignId: String
     
-    /// the in-app type
-    public let inAppType: IterableInAppType
-    
     /// when to trigger this in-app
     public let trigger: IterableInAppTrigger
     
     /// when to expire this in-app, nil means do not expire
     public let expiresAt: Date?
     
-    /// The content of the inApp message
+    /// The content of the inbox message
     public let content: IterableInAppContent
     
+    /// Whether to save this message to inbox
+    public let saveToInbox: Bool
+    
+    /// Metadata such as title, subtitle etc. needed to display this in-app message in inbox.
+    public let inboxMetadata: IterableInboxMetadata?
+    
     /// Custom Payload for this message.
     public let customPayload: [AnyHashable : Any]?
 
-    /// Whether we have processed this message.
+    /// Whether we have processed the trigger for this message.
+    /// Note: This is internal and not public
+    internal var didProcessTrigger = false
+    
+    /// Whether we have processed 'save to inbox' for this message.
     /// Note: This is internal and not public
-    var processed: Bool = false
+    internal var didProcessInbox = false
```

Comment:
It doesn't look like `didProcessInbox` is used anywhere anymore.. can we remove it?

Reply:
oops. Yes need to remove this.